src/ProductConstructionService/ProductConstructionService.Deployment/Deployer.cs

```diff
@@ -222,33 +237,81 @@ private string GetLogsLink(string revisionName)
 
     private async Task<ProcessExecutionResult> InvokeAzCLI(string[] command)
     {
+        string[] fullCommand = [.. command, .. DefaultAzCliParameters];
```

Comment:
```suggestion
        string[] fullCommand = [.. DefaultAzCliParameters, .. command];
```

I think this might be better so that we can override default parameters